src/eventEmitter/index.js

```diff
@@ -0,0 +1,196 @@
+var EventEmitter;
+
+function KuzzleEventEmitter(eventTimeout) {
+  Object.defineProperties(this, {
+    eventTimeout: {
+      value: eventTimeout || 200,
+      writeable: false
+    }
+  });
+
+  if (typeof window !== 'undefined') {
+    this._events = {};
+    this._onceEvents = {};
+  }
+  this.removeAllListeners();
+}
+
+if (typeof window === 'undefined') {
+  EventEmitter = require('events');
+  KuzzleEventEmitter.prototype = new EventEmitter();
```

Comment:
Oh yes, I had an eslint error with 
```javascript
KuzzleEventEmitter.prototype = new require('events')();
```
but I didn't think to try with the parenthesis :/